OWNERS_ALIASES

```diff
@@ -39,7 +39,6 @@ aliases:
     - hogepodge
     - jagosan
   sig-cluster-lifecycle-leads:
-    - roberthbailey
```

Comment:
The OWNERS_ALIASES file here is supposed to be a mirror/copy of the one in the community repo so I wouldn't remove it right away. :)

https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES#L45

Reply:
Roger. I'll undo this one for now.